src/main/java/seedu/address/logic/commands/DeleteStudentFromTutorialCommand.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.NavigationMode;
+import seedu.address.model.student.Student;
+import seedu.address.model.tutorial.Tutorial;
+
+/**
+ * Deletes students from a tutorial slot.
+ */
+public class DeleteStudentFromTutorialCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-student";
+
+    public static final String MESSAGE_USAGE = "Usage: tutorial delete-student TUTORIAL_NAME s/STUDENT_INDEX";
+
+    public static final String MESSAGE_SUCCESS = "Students removed from tutorial!";
+
+    public static final String MESSAGE_TUTORIAL_NOT_FOUND = "Tutorial not found";
+    public static final String MESSAGE_STUDENT_NOT_FOUND = "Student not found";
+
+    private final List<Index> indices;
+    private final Tutorial tutorial;
+
+    /**
+     * Creates an {@link DeleteStudentFromTutorialCommand} to add the specified
+     * {@code Tutorial}
+     */
+    public DeleteStudentFromTutorialCommand(List<Index> indices, Tutorial tutorial) {
+        requireNonNull(indices);
+        requireNonNull(tutorial);
+
+        this.indices = indices;
+        this.tutorial = tutorial;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        if (!model.hasTutorial(tutorial)) {
+            throw new CommandException(MESSAGE_TUTORIAL_NOT_FOUND);
+        }
+
+        List<Student> lastShownList = model.getFilteredStudentList();
+
+        for (Index index : indices) {
+            // Check that index is in bounds.
+            if (index.getZeroBased() >= lastShownList.size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            }
+
+            Student studentToEdit = lastShownList.get(index.getZeroBased());
+            Student editedStudent = studentToEdit.clone();
+            Set<Tutorial> tutorials = new HashSet<>(studentToEdit.getTutorials());
+            tutorials.remove(tutorial);
+            editedStudent.setTutorials(tutorials);
+
+            model.setStudent(studentToEdit, editedStudent);
+        }
+
+        return new CommandResult(MESSAGE_SUCCESS, NavigationMode.TUTORIAL);
```

Comment:
Would this make sense to show `STUDENT` view?

Reply:
I haven't investigated the source code on UIs and viewing but if we don't change the view, will the stale info be displayed to the user? Ideally there should be be a way to refresh the view while keeping whatever predicate/filter that is currently on

Reply:
The way the UI works is quite amazing, even if u don't change the view the information will always be updated, because we're using an ObservableList to store the students

Reply:
```suggestion
        return new CommandResult(MESSAGE_SUCCESS, NavigationMode.UNCHANGED);
```
Can use `UNCHANGED` mode for now, will enforce calling the command from student view via #90 for adding/deleting students from tutorial.